data/mods/Xedra_Evolved/spells/dreamer_spells.json

```diff
@@ -524,8 +524,32 @@
     "shape": "blast",
     "energy_source": "MANA",
     "base_energy_cost": 1000,
-    "base_casting_time": { "math": [ "spell_time(u_val('time: 4 h'))" ] },
-    "final_casting_time": { "math": [ "spell_time(u_val('time: 4 h'))" ] },
+    "base_casting_time": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
+    "final_casting_time": { "math": [ "spell_time(u_val('time: 2 h'))" ] },
     "spell_class": "DREAMER"
+  },
+  {
+    "id": "spell_karma_arms",
+    "type": "SPELL",
+    "name": "Karma Arms",
+    "description": "A four transparent arms grow out of your body.  You have a full control over them, and who knows what power they contain.",
+    "//": "obvious reference for submachine, albeit no portals",
```

Comment:
What does this mean 🤔

Reply:
<https://submachine.fandom.com/wiki/Submachine_Wiki>

Reply:
```suggestion
    "//": "obvious reference for "Submachine" by Mateusz Skutnik, albeit no portals",
```